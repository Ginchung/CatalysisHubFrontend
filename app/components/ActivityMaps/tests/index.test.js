// import React from 'react';
// import { shallow } from 'enzyme';

// import ActivityMaps from '../index';

describe('<ActivityMaps />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
