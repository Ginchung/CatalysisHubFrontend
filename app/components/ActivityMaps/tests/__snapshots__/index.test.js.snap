// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActivityMaps /> should render a flexbox 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Styled(Component)
    display="flex"
    element="div"
    flexDirection="column"
    minHeight="100vh"
  >
    <Styled(Component)
      display="flex"
      element="div"
      height="75px"
      style={
        Object {
          "paddingLeft": "45px",
        }
      }
    >
      <SelectField
        autoWidth={false}
        disabled={false}
        floatingLabelStyle={
          Object {
            "fontFamily": "Arial",
          }
        }
        floatingLabelText="Chemical Reaction"
        fullWidth={false}
        multiple={false}
      >
        <MenuItem
          anchorOrigin={
            Object {
              "horizontal": "right",
              "vertical": "top",
            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          primaryText="HER (2 structures)"
          targetOrigin={
            Object {
              "horizontal": "left",
              "vertical": "top",
            }
          }
          value="HER"
        />
        <MenuItem
          anchorOrigin={
            Object {
              "horizontal": "right",
              "vertical": "top",
            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          primaryText="OER (2 descriptors)"
          targetOrigin={
            Object {
              "horizontal": "left",
              "vertical": "top",
            }
          }
          value="OER"
        />
        <MenuItem
          anchorOrigin={
            Object {
              "horizontal": "right",
              "vertical": "top",
            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          primaryText="ORR (2 descriptors)"
          targetOrigin={
            Object {
              "horizontal": "left",
              "vertical": "top",
            }
          }
          value="ORR"
        />
        <MenuItem
          anchorOrigin={
            Object {
              "horizontal": "right",
              "vertical": "top",
            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          primaryText="CO reduction (2 descriptors)"
          targetOrigin={
            Object {
              "horizontal": "left",
              "vertical": "top",
            }
          }
          value="CO2RR"
        />
        <MenuItem
          anchorOrigin={
            Object {
              "horizontal": "right",
              "vertical": "top",
            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          primaryText="Ammonia Synthesis"
          targetOrigin={
            Object {
              "horizontal": "left",
              "vertical": "top",
            }
          }
          value="Ammonia"
        />
        <MenuItem
          anchorOrigin={
            Object {
              "horizontal": "right",
              "vertical": "top",
            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          primaryText="CO oxidation (2 descriptors)"
          targetOrigin={
            Object {
              "horizontal": "left",
              "vertical": "top",
            }
          }
          value="COx"
        />
        <MenuItem
          anchorOrigin={
            Object {
              "horizontal": "right",
              "vertical": "top",
            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          primaryText="NO reduction (2 descriptors)"
          targetOrigin={
            Object {
              "horizontal": "left",
              "vertical": "top",
            }
          }
          value="deNOx"
        />
      </SelectField>
    </Styled(Component)>
    <Styled(Component)
      display="flex"
      element="div"
      height="500px"
    >
      <Connect(PeriodicTableSelector) />
    </Styled(Component)>
    <div>
      <Styled(Component)
        display="flex"
        element="div"
        height="950px"
        width="100%"
      >
        <ActivityMapOer />
      </Styled(Component)>
      <Styled(Component)
        display="flex"
        element="div"
        height="1000px"
      >
        <StructureView2 />
      </Styled(Component)>
    </div>
  </Styled(Component)>,
  "nodes": Array [
    <Styled(Component)
      display="flex"
      element="div"
      flexDirection="column"
      minHeight="100vh"
    >
      <Styled(Component)
        display="flex"
        element="div"
        height="75px"
        style={
          Object {
            "paddingLeft": "45px",
          }
        }
      >
        <SelectField
          autoWidth={false}
          disabled={false}
          floatingLabelStyle={
            Object {
              "fontFamily": "Arial",
            }
          }
          floatingLabelText="Chemical Reaction"
          fullWidth={false}
          multiple={false}
        >
          <MenuItem
            anchorOrigin={
              Object {
                "horizontal": "right",
                "vertical": "top",
              }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            primaryText="HER (2 structures)"
            targetOrigin={
              Object {
                "horizontal": "left",
                "vertical": "top",
              }
            }
            value="HER"
          />
          <MenuItem
            anchorOrigin={
              Object {
                "horizontal": "right",
                "vertical": "top",
              }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            primaryText="OER (2 descriptors)"
            targetOrigin={
              Object {
                "horizontal": "left",
                "vertical": "top",
              }
            }
            value="OER"
          />
          <MenuItem
            anchorOrigin={
              Object {
                "horizontal": "right",
                "vertical": "top",
              }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            primaryText="ORR (2 descriptors)"
            targetOrigin={
              Object {
                "horizontal": "left",
                "vertical": "top",
              }
            }
            value="ORR"
          />
          <MenuItem
            anchorOrigin={
              Object {
                "horizontal": "right",
                "vertical": "top",
              }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            primaryText="CO reduction (2 descriptors)"
            targetOrigin={
              Object {
                "horizontal": "left",
                "vertical": "top",
              }
            }
            value="CO2RR"
          />
          <MenuItem
            anchorOrigin={
              Object {
                "horizontal": "right",
                "vertical": "top",
              }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            primaryText="Ammonia Synthesis"
            targetOrigin={
              Object {
                "horizontal": "left",
                "vertical": "top",
              }
            }
            value="Ammonia"
          />
          <MenuItem
            anchorOrigin={
              Object {
                "horizontal": "right",
                "vertical": "top",
              }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            primaryText="CO oxidation (2 descriptors)"
            targetOrigin={
              Object {
                "horizontal": "left",
                "vertical": "top",
              }
            }
            value="COx"
          />
          <MenuItem
            anchorOrigin={
              Object {
                "horizontal": "right",
                "vertical": "top",
              }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            primaryText="NO reduction (2 descriptors)"
            targetOrigin={
              Object {
                "horizontal": "left",
                "vertical": "top",
              }
            }
            value="deNOx"
          />
        </SelectField>
      </Styled(Component)>
      <Styled(Component)
        display="flex"
        element="div"
        height="500px"
      >
        <Connect(PeriodicTableSelector) />
      </Styled(Component)>
      <div>
        <Styled(Component)
          display="flex"
          element="div"
          height="950px"
          width="100%"
        >
          <ActivityMapOer />
        </Styled(Component)>
        <Styled(Component)
          display="flex"
          element="div"
          height="1000px"
        >
          <StructureView2 />
        </Styled(Component)>
      </div>
    </Styled(Component)>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <ActivityMaps />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": ActivityMaps {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Styled(Component)
          display="flex"
          element="div"
          flexDirection="column"
          minHeight="100vh"
        >
          <Styled(Component)
            display="flex"
            element="div"
            height="75px"
            style={
              Object {
                "paddingLeft": "45px",
              }
            }
          >
            <SelectField
              autoWidth={false}
              disabled={false}
              floatingLabelStyle={
                Object {
                  "fontFamily": "Arial",
                }
              }
              floatingLabelText="Chemical Reaction"
              fullWidth={false}
              multiple={false}
            >
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="HER (2 structures)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="HER"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="OER (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="OER"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="ORR (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="ORR"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="CO reduction (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="CO2RR"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="Ammonia Synthesis"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="Ammonia"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="CO oxidation (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="COx"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="NO reduction (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="deNOx"
              />
            </SelectField>
          </Styled(Component)>
          <Styled(Component)
            display="flex"
            element="div"
            height="500px"
          >
            <Connect(PeriodicTableSelector) />
          </Styled(Component)>
          <div>
            <Styled(Component)
              display="flex"
              element="div"
              height="950px"
              width="100%"
            >
              <ActivityMapOer />
            </Styled(Component)>
            <Styled(Component)
              display="flex"
              element="div"
              height="1000px"
            >
              <StructureView2 />
            </Styled(Component)>
          </div>
        </Styled(Component)>,
        "_debugID": 2,
        "_renderedOutput": <Styled(Component)
          display="flex"
          element="div"
          flexDirection="column"
          minHeight="100vh"
        >
          <Styled(Component)
            display="flex"
            element="div"
            height="75px"
            style={
              Object {
                "paddingLeft": "45px",
              }
            }
          >
            <SelectField
              autoWidth={false}
              disabled={false}
              floatingLabelStyle={
                Object {
                  "fontFamily": "Arial",
                }
              }
              floatingLabelText="Chemical Reaction"
              fullWidth={false}
              multiple={false}
            >
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="HER (2 structures)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="HER"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="OER (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="OER"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="ORR (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="ORR"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="CO reduction (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="CO2RR"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="Ammonia Synthesis"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="Ammonia"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="CO oxidation (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="COx"
              />
              <MenuItem
                anchorOrigin={
                  Object {
                    "horizontal": "right",
                    "vertical": "top",
                  }
                }
                checked={false}
                desktop={false}
                disabled={false}
                focusState="none"
                insetChildren={false}
                primaryText="NO reduction (2 descriptors)"
                targetOrigin={
                  Object {
                    "horizontal": "left",
                    "vertical": "top",
                  }
                }
                value="deNOx"
              />
            </SelectField>
          </Styled(Component)>
          <Styled(Component)
            display="flex"
            element="div"
            height="500px"
          >
            <Connect(PeriodicTableSelector) />
          </Styled(Component)>
          <div>
            <Styled(Component)
              display="flex"
              element="div"
              height="950px"
              width="100%"
            >
              <ActivityMapOer />
            </Styled(Component)>
            <Styled(Component)
              display="flex"
              element="div"
              height="1000px"
            >
              <StructureView2 />
            </Styled(Component)>
          </div>
        </Styled(Component)>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ActivityMaps />,
}
`;
